code/modules/mob/new_player/preferences_setup.dm

```diff
@@ -644,7 +644,7 @@
 						clothes_s.Blend(new /icon('icons/mob/back.dmi', "satchel-med"), ICON_OVERLAY)
 					if(4)
 						clothes_s.Blend(new /icon('icons/mob/back.dmi', "satchel"), ICON_OVERLAY)
-			if(DOCTOR)
+			if(DOCTOR || SENIORMED)

```

Comment:
Not related to the PR, but we should really port /tg/ and bay's update to get rid of this awful list.